modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,39 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[source,terminal]
+----
+$ rosa delete operator-roles -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto --yes <1>
+----
+<1> Replace `<prefix>` with the prefix of the account roles that are being deleted, for example `ManagedOpenShift`.
++
+[IMPORTANT]
+====
+Account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the resources if they are not required by other clusters.
```

Comment:
Do we need this important note twice?

Reply:
I originally did not have this important note twice in this section. We added it here as well, for the reasons outlined in [this conversation](https://github.com/openshift/openshift-docs/pull/39632#discussion_r764816742) with @vkareh.

Reply:
Following a conversation with @0kashi, I am leaving both of the admonitions in place.